pkg/controller/networkpolicy/validate.go

```diff
@@ -726,3 +799,33 @@ func (g *groupValidator) deleteValidate(oldObj interface{}, userInfo authenticat
 	}
 	return "", true
 }
+
+func (g *groupValidator) deleteValidateGroup(oldG *crdv1alpha3.Group) (string, bool) {
```

Comment:
I don't think we should add these back now that we are removing the same constraints for CG

Reply:
this too will disappear once we merge your PR